docs/spark/how-to-guides/connect-to-azure-storage.md

```diff
@@ -0,0 +1,74 @@
+    Your final `hadoop-env.cmd` file should look something like this:
+    ![Final hadoop-env.cmd file](./media/connect-external-sources/hadoop-env.png)
+
+## Steps to configure Hadoop to connect to your Storage account
+
+1. Open the ADLS Gen 2 or WASB storage account you want to connect to through the [Azure portal](https://portal.azure.com) and open the **Access keys** panel under the **Settings** blade and copy the value of **Key** from under key1.
+2. Now in order to configure Hadoop to access your ADLS Gen2 account you would have to edit your `core-site.xml` (located in `%HADOOP_HOME%\etc\hadoop\` ) file which contains cluster-wide configuration. Add the following properties inside the `<configuration>` tags in this file:
+
+    ```xml
+    <configuration>
+      <property>
+        <name>fs.azure.account.auth.type.YOUR_ACCOUNT_NAME.dfs.core.windows.net</name>
+        <value>SharedKey</value>
+        <description></description>
+      </property>
+      <property>
+        <name>fs.azure.account.key.YOUR_ACCOUNT_NAME.dfs.core.windows.net</name>
+        <value>YOUR ACCESS KEY (copied from Step 1)</value>
+        <description>The secret password. Never share these.</description>
+      </property>
+    </configuration>
+    ```
+
+3. If you are trying to connect to a WASB account, replace `dfs` with `blob` in the property names, for example, `fs.azure.account.auth.type.YOUR_ACCOUNT_NAME.blob.core.windows.net`.
+4. You can test the connectivity to your Storage Account from Hadoop by running the following command from your `%HADOOP_HOME%` directory:
+
+  ```bash
+  bin\hdfs dfs -ls <URI to your account>
+  ```
+
+This should display a list of all files/folders in the path provided by your URI.
+
+> NOTE: The format to derive the URI to your Storage account is as follows:
+
+```
+For ADLS: abfs[s]://<file_system>@<account_name>.dfs.core.windows.net/<path>/<file_name>
+For WASB: wasb[s]://<file_system>@<account_name>.blob.core.windows.net/<path>/<file_name>
+```
+
+## Connect to the account using local .NET for Apache Spark instance
+
+1. If the above command worked, you can now move on to accessing this storage account through Spark. First run the command `hadoop classpath` from the commandline inside `%HADOOP_HOME%` and copy the output.
+2. Set the output of the command run in Step 1 to the value of environment variable `SPARK_DIST_CLASSPATH`.
+3. Now you should be able to access your ADLS or WASB storage account through Spark .NET using the abfs URI as shown in the simple example below:
+
+  ```csharp
+  SparkSession spark = SparkSession
+      .Builder()
+      .AppName("Connect to Azure Storage locally")
+      .GetOrCreate();
+  DataFrame df = spark.Read().Json("wasbs://file_system@account_name.blob.core.windows.net/path/file.json");
+  //DataFrame df = spark.Read().Json("abfss://file_system@account_name.dfs.core.windows.net/path/file.json");
+  df.Show();
+  ```
```

Comment:
The code block currently doesn't show **inside** the list item. This should fix that.

~~~suggestion
   ```csharp
   SparkSession spark = SparkSession
       .Builder()
       .AppName("Connect to Azure Storage locally")
       .GetOrCreate();
   DataFrame df = spark.Read().Json("wasbs://file_system@account_name.blob.core.windows.net/path/file.json");
   //DataFrame df = spark.Read().Json("abfss://file_system@account_name.dfs.core.windows.net/path/file.json");
   df.Show();
   ```
~~~